test/inductor/test_static_cuda_launcher.py

```diff
@@ -1,13 +1,12 @@
 # Owner(s): ["module: inductor"]
```

Comment:
I went ahead and revamped the testing setup so we don't have to manually call save_to_tmp_file() each time, instead opting to set _cubin_path similar to how `triton_heuristics` handles it. I think it's cleaner this way. No changes in behavior to existing tests.